app/models/organization.rb

```diff
@@ -1,4 +1,7 @@
 class Organization < ApplicationRecord
-  has_many :cities_organizations
+  has_many :cities_organizations, dependent: :destroy
   has_many :cities, through: :cities_organizations
+
+  validates :adress, :email, :name, :phone_number, presence: true
```

Comment:
You should be able to combine these two into one
`validates :adress, :email, :name, :phone_number, presence: true, uniqueness: true`